src/app/components/Menu.jsx

```diff
@@ -48,16 +49,30 @@ const Menu = ({ location }) => {
         setSandboxOutdatedData({ isOutdated: true, latestVersion });
       }
     });
+  }, []);
 
+  useEffect(() => {
     if (location.pathname.includes(NAMED_ROUTES.LIBRARY_EDITOR)) {
       setSelectedKeys(new Set([NAMED_ROUTES.LIBRARY_EDITOR]));
     } else if (location.pathname.endsWith(NAMED_ROUTES.LIB_SANDBOX)) {
       setSelectedKeys(new Set([NAMED_ROUTES.LIB_SANDBOX]));
     } else if (location.pathname.includes(NAMED_ROUTES.VIEW_SANDBOX)) {
       setSelectedKeys(new Set([NAMED_ROUTES.VIEW_SANDBOX]));
+    } else {
+      setSelectedKeys(new Set([]));
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [location.pathname]);
+
+  const { platform } = useContext(ExtensionDescriptorContext);
+  let menuItems = [
+    { key: NAMED_ROUTES.VIEW_SANDBOX, textValue: 'View Sandbox', Icon: ViewSandboxIcon }
+  ];
+  if (platform !== PLATFORMS.MOBILE) {
```

Comment:
Is the lib sandbox useful for `edge` extensions or should we hide it for those as well?